frontend/src/scenes/session-recordings/player/metaLogic.test.ts

```diff
@@ -35,7 +42,7 @@ describe('metaLogic', () => {
         it('stops loading after meta load is successful', async () => {
             await expectLogic(logic).toMatchValues({ loading: true })
             await expectLogic(logic, () => {
-                logic.actions.loadRecordingMetaSuccess({} as SessionPlayerData)
+                logic.connections['scenes.session-recordings.sessionRecordingLogic'].actions.loadRecordingMeta('1')
```

Comment:
This feels pretty hacky - not sure if there's a better way of doing it.

Before, this used to call the `loadRecordingMetaSuccess` action, but that has a bunch of side-effects that depend on the results of `loadRecordingMeta`. This was the cause of the errors we were seeing.

To resolve the errors and keep the test, I updated the test to call `loadRecordingMeta`, but it feels kind of messy...

Meta point: this test isn't testing much. Might make sense to just delete the test and get rid of the headache?